{
  try {
    lockOn();
  }
 catch (  InterruptedException ex) {
  }
}
